docs/index.rst

```diff
@@ -3,26 +3,37 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+========
 Welcome!
 ========
 
-Welcome to the Contributors Portal for Access to Memory (AtoM), Archivematica and Enduro.
+Welcome to the Contributors Portal for Access to Memory (AtoM),
+Archivematica and Enduro.
+
+At `Artefactual <https://www.artefactual.com>`__, we are currently
+working on improving the contribution process, and we deemed useful
```

Comment:
- "we deemed it useful" (need the "it" in there)
- "to have a Portal to group relevant information" (not regroup)